java/ql/lib/semmle/code/java/security/CommandLineQuery.qll

```diff
@@ -58,6 +58,13 @@ module InputToArgumentToExecFlowConfig implements DataFlow::ConfigSig {
   predicate isAdditionalFlowStep(DataFlow::Node n1, DataFlow::Node n2) {
     any(CommandInjectionAdditionalTaintStep s).step(n1, n2)
   }
+
+  // It's valid to use diff-informed data flow for this configuration because
+  // the location of the selected element in the query is contained inside the
+  // location of the sink. The query, as a predicate, is used negated in
```

Comment:
Does containment in another location actually guarantee anything, given that alert-filtering only looks at the start line.

Reply:
No, I had misunderstood how this works. I'm working on a follow-up PR to make this configuration exact.